### ISSUE TITLE
Aligning the "Help" and "Support" wording in our pages

### ISSUE THREAD

**Issue by correliebre** [Automattic/wp-calypso#93125]
### Quick summary

From @eurello: "We should use "help" for documentation and "support" for contacting support. Basically if someone is trying to contact support, let's show them the route to get there and call it "support" or "contact." But we shouldn't show a link to "support" if it takes them to a knowledge base and then they have to find support somewhere on that screen. Then we should link to "help.""

### Steps to reproduce

In wordpress.com/support, at the top, the button should be changed from "Get help" to "Get support"
<img width="1105" alt="Screenshot 2024-07-30 at 7 38 26 PM" src="https://github.com/user-attachments/assets/6dee05d0-9fea-4fd6-bccc-296b1580194a">

In wordpress.com/support, at the bottom, the button should be changed from "Get help" to "Get support". 
<img width="1062" alt="Screenshot 2024-07-30 at 7 38 41 PM" src="https://github.com/user-attachments/assets/58434ba3-0a10-493c-9138-a90c4cadf45b">

@eurello could you confirm whether this one should also be changed? I think it could be "Still need support?", but reading "Still need help?" also makes sense given that the user was already in a space offering help. The next step here takes them to Wapuu already.
<img width="472" alt="Screenshot 2024-07-30 at 7 36 18 PM" src="https://github.com/user-attachments/assets/c43a7dc3-a7b9-4f4e-8f90-543c8d50c05b">

### What you expected to happen

-

### What actually happened

-

### Impact

All

### Available workarounds?

No but the platform is still usable

**Comment by eurello** [comment]
@correliebre I think I misled you here in what the preferred labels are -- what's there now is actually fine, and has already been addressed, see this closed issue here: https://github.com/Automattic/dotcom-forge/issues/8122#issuecomment-2220553124

The issue was that "contact support" leads customers to think that it will open the contact option immediately. Instead, it opens Wapuu, which feels confusing. So we fixed that already. I got confused when you were asking me about this, and also forgot that it had been fixed. Sorry!
